content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,48 +1,109 @@
 
-Enable ticket closing
-=====================
+Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
+Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
 
-Start by navigating to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and choose
-the appropriate team. Then click :guilabel:`Edit` and enable :guilabel:`Ticket closing` by
-checking the field box.
+If one of the team's stages is set to be folded in the kanban view, it will be the default selection
+in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that
+occurs first in the pipeline will be the default. If no stage is folded, the default selection will
+be the last stage in the pipeline.
 
-.. image:: close_tickets/close-ticket-settings.png
-   :alt: Ticket closing feature in Odoo Helpdesk.
+The :guilabel:`After days of inactivity` field defaults to `7`, but can be adjusted if necessary.
 
-To designate which stage the ticket migrates to once it is closed, navigate to the ticket pipeline
-by going to :menuselection:`Helpdesk --> Overview` and clicking :guilabel:`Tickets` on the team's
-card.
+.. warning::
+   The :guilabel:`After days of inactivity` field does **not** take the working calendar into
+   account when tracking the amount of time a ticket has been inactive.
 
-There are two options: create a new Kanban stage or work with an existing one. For both scenarios,
-click the :guilabel:`Settings (gear)` icon next to the stage name, select :guilabel:`Edit Stage`,
-and enable :guilabel:`Closing Stage`. After checking the field box, click :guilabel:`Save` to
-finish.
+If only certain stages should be used to track days of inactivity, they can be added to the
+:guilabel:`In Stages` field.
 
-.. note::
-   If a closing stage is not specified, by default, the ticket is moved to the last stage in the
-   kanban. If more than one stage is set as a closing stage, the ticket is placed in the first
-   closing stage column.
+.. example::
+   A team's pipeline is created with the following stages:
 
-With the ticket closing settings now complete, customers can now view the option to
-:guilabel:`Close this ticket` when they log into their portal.
+   - `New`
+   - `In Progress`
+   - `Customer Feedback`
+   - `Closed`
 
-.. image:: close_tickets/customer-view-close-ticket.png
-   :alt: Customer view of ticket closing in Odoo Helpdesk.
+   Tickets can linger in the :guilabel:`Customer Feedback stage`, because once an issue is solved,
+   customers may not respond immediately. At that point, the tickets can be closed automatically.
+   However, tickets in the :guilabel:`New` and :guilabel:`In Progress` stages may remain inactive
+   due to assignment or workload issues. Closing these tickets automatically would result in issues
+   going unsolved.
+
+   Therefore, the :guilabel:`Automatic Closing` settings would be configured as below\:\
```

Comment:
Images are only supplemental to the writing; the writing needs to stand on it's own.
Actually write this part out 😉 

```suggestion
   Therefore, the :guilabel:`Automatic Closing` settings would be configured as below\:\
      
   - :guilabel:`Automatic Closing`: *checked*
   - :guilabel:`Move to Stage`: `Solved`
   - :guilabel:`After``7`:guilabel:`days of inactivity`
   - :guilabel:`In Stages`: `Customer Feedback`
```